shed/event_streams.py

```diff
@@ -464,7 +464,16 @@ def issue_event(self, outputs):
                                        for (output_name, desc), output in
                                        zzip(self.output_info, outputs)})
             else:
-                new_event.update(data=outputs['data'])
+                if outputs is None:
+                    outputs = {}
+                elif not isinstance(outputs, dict):
+                    errormsg = "Error, outputs is not a dict. Can't continue"
```

Comment:
yeah thanks. the reason why i don't do it that way is i don't like how we lose the indent. if we can find a way without that would be great. but if you prefer this way im still fine with it